### ISSUE TITLE
[deckhouse] actual status on pending release that would be skipped

### ISSUE THREAD

**Issue by yalosev** [deckhouse/deckhouse#6189]
### Preflight Checklist

- [x] I agree to follow the [Code of Conduct](https://github.com/deckhouse/deckhouse/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/deckhouse/deckhouse/issues) for an issue that matches the one I want to file, without success.

### Use case. Why is this important?

```
v1.51.6    Deployed      2d1h            
v1.52.4    Pending      2d1h             Release is postponed until: 11 Oct 23 09:49 UTC
v1.52.5    Pending      44h              Release is postponed until: 11 Oct 23 15:24 UTC
v1.52.6    Pending      53m              Release is postponed until: 13 Oct 23 10:41 UTC
```

this status is very confusing. I assume we can make it like
```
v1.51.6    Deployed      2d1h            
v1.52.4    Pending      2d1h              Would be skipped
v1.52.5    Pending      44h               Would be skipped
v1.52.6    Pending      53m              Release is postponed until: 13 Oct 23 10:41 UTC
```

### Proposed Solution

We can update a release status also [here](https://github.com/deckhouse/deckhouse/blob/main/modules/002-deckhouse/hooks/internal/updater/updater.go#L492)
Set it to `Would be skipped`
After the deploying process, when release will skipped, this status would set to `""`

**Comment by konstantin-axenov** [comment]
Maybe it's better to delete `Pending` releases that won't be installed.

**Comment by shvgn** [comment]
> Maybe it's better to delete Pending releases that won't be installed.

Unfortunately, a part of changelogs will be missing in cluster. UX-wise it is blocked by https://github.com/deckhouse/deckhouse/issues/4391

**Comment by shvgn** [comment]
We already have such an issue e.g. with stable release channel which virtually never receive .0 patch versions

**Comment by deckhouse-BOaTswain** [comment]
This issue has been automatically put in the triage queue because it has not had
recent activity. The team will reconsider the status of this issue. Thank you
for your contributions.
